ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -153,10 +153,10 @@ objects:
 
         i=0
         MAX_RETRIES=10
-        until oc wait "${ROUTER_DEPLOYMENT}" -n "${ROUTER_NAMESPACE}" --for condition=available --timeout=10m || [ $i -eq $MAX_RETRIES ]; do
+        until oc rollout status "${ROUTER_DEPLOYMENT}" -n "${ROUTER_NAMESPACE}" -w || [ $i -eq $MAX_RETRIES ]; do
```

Comment:
I'm not clear on how `-w` plays with `MAX_RETRIES` and the `sleep 60` at the tail of the loop.  But a max of 10 infinitely-blocking calls seems like something that's going to hang for too long sometimes ;).  Maybe something like:

```sh
i=0
MAX_RETRIES=10
TARGET="$(date +%s)"
TARGET="$((TARGET + 60))"
until oc --request-timeout=55s oc rollout status "${ROUTER_DEPLOYMENT}" -n "${ROUTER_NAMESPACE}" -w || [ $i -eq $MAX_RETRIES ]; do
  i=$((i+1))
  ...
  REMAINING="$((TARGET - NOW))"
  sleep "${REMAINING}"
  TARGET="$((TARGET + 60))"
done
```

Reply:
lol - yeah that's true but figured its going to fail anyway beyond this point if we don't get back the rollout info (assume there is a timeout above this in the caller) but ... fair point. I created this PR as an alternative - am not certain its needed as @smarterclayton  was ok with the other alternative in #2001 - putting this one on hold.

Reply:
I think having a few retries might be good to do anyway to handle any other connectivity/spurious issues. I'll fix this up as per your original comments with a timeout. Thanks.